.github/renovate.json

```diff
@@ -30,7 +30,6 @@
       ],
       "matchCurrentVersion": "!/^0/",
       "matchDepNames": [
-        "!app",
```

Comment:
Why this change?

Reply:
Because the example packages shouldn't be treated like internal packages where we always update to the latest version.

Reply:
but shouldn't we always use the latest version for all our examples?
1. verify that our examples still work with the latest versions.
2. with this being the app we ship to end users, I'd like to make sure to pull in all available fixes

Reply:
Hm ok makes sense